jl4-lsp/src/LSP/L4/Viz/VizExpr.hs

```diff
@@ -133,7 +135,11 @@ instance HasCodec RenderAsLadderInfo where
     named "RenderAsLadderInfo" $
       object "RenderAsLadderInfo" $
         MkRenderAsLadderInfo
-          <$> requiredField' "funDecl" .= view #funDecl
+          <$> requiredField' "verTextDocId" .= view #verTextDocId
+          <*> requiredField' "funDecl"      .= view #funDecl
+
+instance HasCodec LSP.VersionedTextDocumentIdentifier where
+  codec = codecViaAeson "VersionedTextDocumentIdentifier"
```

Comment:
can you inline this instances into the instances you need it in? then we could skip the orphan ~ that's also nicer in the sense that we don't *use* this codec for ser/de of VersionedTextDocumentIdentifier?

Reply:
I looked briefly into trying to inline the instance, but I think it's fine to stick with this for the time being (The autodocodec piggybacks on the aeson instance, so I don't think there's a real risk of doing some kind of unorthodox serializing/deserializing no matter what.). The orphan instance is not great, but that will be fixed if we switch away from autocodec back to aeson in the future.